libs/core/langchain_core/language_models/base.py

```diff
@@ -71,6 +78,28 @@ class BaseLanguageModel(
     All language model wrappers inherit from BaseLanguageModel.
     """
 
+    cache: Optional[bool] = None
+    """Whether to cache the response."""
+    verbose: bool = Field(default_factory=_get_verbosity)
+    """Whether to print out response text."""
+    callbacks: Callbacks = Field(default=None, exclude=True)
+    """Callbacks to add to the run trace."""
+    tags: Optional[List[str]] = Field(default=None, exclude=True)
+    """Tags to add to the run trace."""
+    metadata: Optional[Dict[str, Any]] = Field(default=None, exclude=True)
+    """Metadata to add to the run trace."""
+
+    @validator("verbose", pre=True, always=True)
+    def set_verbose(cls, verbose: Optional[bool]) -> bool:
```

Comment:
was inconsistent before - this used to only be on llms but not chat model. Will now have same behavior for `verbose=None` for both